src/sentry/options/defaults.py

```diff
@@ -52,11 +52,7 @@
 register("system.logging-format", default=LoggingFormat.HUMAN, flags=FLAG_NOSTORE)
 # This is used for the chunk upload endpoint
 register("system.upload-url-prefix", flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE)
-register(
-    "system.maximum-file-size",
```

Comment:
Did you already delete this option in the automator?

Reply:
Thank you for reminding me of this.
Seems like this was only defined for S4S, removing it here: https://github.com/getsentry/sentry-options-automator/pull/3584